packages/paste-core/components/combobox/__tests__/index.spec.tsx

```diff
@@ -208,6 +233,21 @@ describe('Combobox', () => {
       const renderedTextbox = screen.getByRole('textbox');
       expect(renderedTextbox.getAttribute('required')).toEqual('');
     });
+
+    it('should call scroll function when highlighted index changes', () => {
+      render(<ComboboxMock />, {wrapper: ThemeWrapper});
+
+      const targetIndex = 1;
+      const target = items[targetIndex];
+
+      userEvent.hover(screen.getByText(target));
```

Comment:
Initial highlighted index is -1. We need to simulate a user event because the index change is coupled to the mouseover event.

These are things internal to the primitive from dropdown.